utils/create_new_app.js

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env node
```

Comment:
+1 to converting this file to JS. Its best to stick with one language/ecosystem, helps with reusing utilities.